### ISSUE TITLE
False negative with nested function

### ISSUE THREAD

**Issue by rhansen** [TristonJ/eslint-plugin-prefer-arrow#24]
The prefer-arrow-functions rule does not complain about `foo` in this example:

```javascript
const foo = function () {
  const bar = function () {
    console.log(this.toString());
  };
  return bar;
};
```

**Comment by TristonJ** [comment]
Closed by #26, thank you!
